index.css

```diff
@@ -482,6 +486,53 @@ html * {
   .full-screen-blur {
     filter: blur(0.3rem);
   }
+
+  .modal {
+    width: 100%;
+    height: 100%;
+    background-color: rgba(255, 255, 255, 0.7);
+    position: fixed;
+    left: 0;
+    top: 0;
+    z-index: 2;
+  }
```

Comment:
- [ ] As requested by the previous reviewer that you should work on the pop-up modal card by following what we have in the Figma design as much as possible. Currently, I notice your image is taking a small part of the card width which is quite different from what we have in the Figma design.

  - The text is not aligning properly inside the card width as it is hitting the bottom of the card. kindly give it some margin or padding-bottom
  - The buttons are not well aligned inside the card as currently, the are moving outside the card in the `desktop version` while in the `mobile version` it has been cut off

Desktop version 👇 

| Your Design 👇  | Figma Design 👇  |
| --- | --- |
| ![hor-card](https://user-images.githubusercontent.com/78634841/209416276-c3c67f08-8ee3-4576-ad42-fa513bb2feed.png)  | ![pp1-real](https://user-images.githubusercontent.com/78634841/209416263-86f51664-3a02-4b89-8dfa-9bc373cfc6c5.png) |

Mobile version 👇 

| Your Design 👇  | Figma Design 👇  |
| --- | --- |
| ![hor-cad](https://user-images.githubusercontent.com/78634841/209416322-02010f2c-f285-4dce-8dbe-949d07cd5a37.png) | ![real-pop](https://user-images.githubusercontent.com/78634841/209416332-50a2a155-5a67-4468-82ae-aa8aba17e551.png) |

- The link to the last card image is broken and it's not showing in the pop-up modal for both mobile and desktop version

Reply:
@brainconnect93  I fixed the height of modal but using the same height and width than figma doesn't work properly. According to figma desing for big screen they are using 1440 x 1577 and for small screens 375 x 823. So I adapted the modal for those widths at start but following the same sizes from template looks impossible to fill the image in that little space. In a default device toolbar inspector there's no way that height starts in the template size. So what can you recommend me? Because following the exact size from template `Does not work`

| Figma Big screen Size  | Figma Small screen Size |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/77707836/209421988-1ac6ed34-602c-47c1-a43d-6228f6e62b14.png)  | ![image](https://user-images.githubusercontent.com/77707836/209422011-16afe0b6-7cd3-46bc-8195-cd8579e7b862.png)  |

| Default device Toolbar  | Figma Modal body size |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/77707836/209422155-53d452a1-a0f7-492c-8abb-e599b05aebb7.png) |  ![image](https://user-images.githubusercontent.com/77707836/209422222-b5dfd282-e63f-4e17-be32-0613ef44d496.png) |